package/yast2-squid.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Nov 26 07:51:22 UTC 2018 - Noah Davis <noahadvs@gmail.com>
+
+- Provide icon with module (boo#1109310)
+
```

Comment:
Add the new version also in the changelog entry.

Like in https://github.com/yast/yast-journal/pull/37/files#diff-de2aa5fdf8fb90e97f87829e5594fe75R5

The same for the rest of modules.